Common/UnitDefinitions/ThermalInsulance.json

```diff
@@ -81,8 +81,8 @@
     {
       "SingularName": "HourSquareFeetDegreeFahrenheitPerBtu",
       "PluralName": "HourSquareFeetDegreesFahrenheitPerBtu",
-      "FromUnitToBaseFunc": "{x} * 176.1121482159839",
-      "FromBaseToUnitFunc": "{x} / 176.1121482159839",
+      "FromUnitToBaseFunc": "{x} * (1000 * 0.3048 * 0.3048 * 3600) / (1055.05585262 * 1.8)",
```

Comment:
Verified.

At first I thought 1.8 should be moved to before division to follow naming, but realized it must be in this form to not be rounded.